src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+        break;
+
+      case 'removeProperties':
+        const arrKeysToRemove = action.keysToRemove;
```

Comment:
arrKeysToRemove is unnecessary here, feel free to remove it and use `action.keysToRemove` instead
P.S. if you find a way to desctructure `action` - it will be just `keysToRemove`